### DISCUSSION TITLE
How to make markor align text to left as default?

### DISCUSSION THREAD

**Discussion by rsyqvthv** [gsantner/markor#2074] [General] [answered]
current everything shows in my markor is align to center, checked all through the settings, I can't find where is any setting for change the align, is that's not possible with markor or I missed somehow?

**Comment by gsantner** [comment]
You unintentionally enabled a Markor settings option. Check settings, Editor section

**Reply by rsyqvthv** [reply]
I am delighted to learn that this feature is customizable. However, could you kindly advise me as to the location of the corresponding configuration option? Regrettably, I seem to lack the acumen to locate it independently.

thanks get back to me, but I still haven't get this done, even I checked every single options in the "edit mode" section. 

anyway, I had reset the app to get align left, and set all the other options through, I thought it will be more easier for me. :)

**Reply by gsantner** [reply, marked as answer]
It is not really hard to miss and it exactly describes what your question is about :-).  
But a data reset works too of course.

 ![Screenshot_20230713-202131_Marder-append](https://github.com/gsantner/markor/assets/6735650/4003c07b-6c0f-43f7-af14-f801eeee14d5)

**Reply by rsyqvthv** [reply]
Thanks @gsantner for your detailed screenshots, very appreciate. 

Yes, I did check this option many times, it didn't work at that time, and also the description underneath seems to be that this option just puts the edited line in the middle of the screen, so it's confused me.

after I reset app and adjust other options, I find that quite a lot of Markor's option won't affect until I restart the app. maybe that's the reason how I missed above settings.

Anyway, I learn more how Markor works now, 
Thanks for your help.
